client/dashboard/components/card/index.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-restricted-imports */
+import {
+	Card,
+	CardBody,
+	CardDivider,
+	CardHeader,
+	CardFooter,
+	CardMedia,
+} from '@wordpress/components';
+/* eslint-enable no-restricted-imports */
+import { useViewportMatch } from '@wordpress/compose';
+import { forwardRef, PropsWithChildren, HTMLAttributes } from 'react';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type CardProps = PropsWithChildren< HTMLAttributes< HTMLDivElement > > & Record< string, any >;
```

Comment:
I think we can more copy the Card's prop type like this

```suggestion
type CardProps = React.ComponentProps< typeof Card >;
```

That way we're not having to mix in a `Record< string, any >` as a catch all.

Reply:
Lovely. 🥇